README.md

```diff
@@ -5,18 +5,17 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 ## Requirements
 - A supported operating system.
 - MySQL or PostgreSQL database in which to store Airflow metadata.
-- [Airflow](https://github.com/teamclairvoyant/apache-airflow-parcels) and [RabbitMQ](https://github.com/teamclairvoyant/rabbitmq-cloudera-parcel) parcels need to be installed.
 
 ### Currently Supported Versions of Airflow
-- Airflow 1.7.1.3
-- Airflow 1.8.0
+- Airflow 1.10
```

Comment:
versions available:
1.10
1.10.1
1.10.3

Reply:
Do we want to update the README every time we build?  I left out patch-level versions just for that purpose.

Reply:
OK that should be fine then